src/cripts/CMakeLists.txt

```diff
@@ -45,8 +45,10 @@ set(CRIPTS_PUBLIC_HEADERS
     ${PROJECT_SOURCE_DIR}/include/cripts/Urls.hpp
     ${PROJECT_SOURCE_DIR}/include/cripts/UUID.hpp
     ${PROJECT_SOURCE_DIR}/include/cripts/Configs.hpp
-    ${PROJECT_SOURCE_DIR}/include/cripts/Bundles/Common.hpp
-    ${PROJECT_SOURCE_DIR}/include/cripts/Bundles/LogsMetrics.hpp
+)
+
+set(CRIPTS_BUNDLE_HEADERS ${PROJECT_SOURCE_DIR}/include/cripts/Bundles/Common.hpp
+                          ${PROJECT_SOURCE_DIR}/include/cripts/Bundles/LogsMetrics.hpp
```

Comment:
Shouldn't the paths be quoted? (also above)
If `PROJECT_SOURCE_DIR` contains spaces and doesn't already have quotes, this will break.

Reply:
They weren't quoted before, and I just moved them.